client/settings.ts

```diff
@@ -32,6 +32,7 @@ export const FALSE_VALUES = ['false', 'FALSE', 'no', 'NO', '0']
 export const METADATA_FORMATS = ['yaml', 'json']
 export const METADATA_TYPES = ['resource', 'dialect', 'schema', 'checklist', 'pipeline']
 export const MAX_TABLE_SOURCE_SIZE = 1000000
+export const PROJECT_SYNC_INTERVAL_MILLIS = 10 * 1000
```

Comment:
Are we having 10 or 5 seconds?

Reply:
Yea, it's 10, updated the PR's description